charts/lagoon-logging/values.yaml

```diff
@@ -251,6 +251,11 @@ enableDefaultForwarding: true
 # Set how many fluentd pods should be running
 fluentdReplicaCount: 3
 
+fluentd:
```

Comment:
It's a pain, but you should probably use the unnested concatenated names here to avoid confusion - eg fluentdTolerations and fluentdAffinity, as per the fluentdReplicaCount and fluentdMetrics